### DISCUSSION TITLE
mod_rewrite里面的QUERY_ADD配置问题

### DISCUSSION THREAD

**Discussion by willzhujian** [bfenetworks/bfe#817] [Q&A (问题和回答)] [answered]
![image](https://user-images.githubusercontent.com/54895023/132473530-bef8aa0e-c888-43bb-bcef-341cf11e1022.png)
加入这一段测试配置后，启动报错
![image](https://user-images.githubusercontent.com/54895023/132473605-fae435ec-c66e-4a05-b27c-46d23bb02862.png)
文档里面没有具体怎么配置

**Comment by iyangsj** [comment, marked as answer]
@willzhujian 
配置文件中的Params参数应写为 ["QueryKey", "QueryValue"]

**Reply by willzhujian** [reply]
我想支持VUE的这种 #/Fund/Index，路由查询方式，好像这种不行呀，怎么弄呀
